src/main/java/uk/gov/hmcts/reform/sscs/job/ProcessingVenueMigrationJob.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.sscs.job;
+
+import static java.time.LocalDateTime.now;
+
+import java.io.IOException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.sscs.services.ProcessingVenueMigrationService;
+import uk.gov.hmcts.reform.sscs.util.CaseLoaderTimerTask;
+
+@Component
+@Slf4j
+public class ProcessingVenueMigrationJob extends SscsJob {
```

Comment:
let's find a way of reusing this class for both interpreter and venue migration. one way is to have a parent class, and two child classes for interpreter and venue, with the only difference being the start and end hour for each one
they should probably have a field/method for the consumer that's gonna be used later on in CcdCasesSender (see comment below)